schemas/reservationSchema.js

```diff
@@ -0,0 +1,69 @@
+const Joi = require('joi')
+const mongoose = require('mongoose')
+
+const reservationSchema = new mongoose.Schema({
+  userId: {
+    type: mongoose.Types.ObjectId,
+    ref: 'User',
+    required: true,
+  },
+  hotelId: {
+    type: mongoose.Types.ObjectId,
+    ref: 'Hotel',
+    required: true,
+  },
+  roomlId: {
+    type: mongoose.Types.ObjectId,
+    ref: 'Room',
+    required: true,
+  },
+  startDate: {
+    type: Date,
+    required: true,
+  },
+  endDate: {
+    type: Date,
+    required: true,
+  },
+  people: {
+    type: {
+      adults: {
+        type: Number,
+        required: true,
+        min: 1,
+      },
+      children: {
+        type: Number,
+        required: true,
+        min: 0,
+      },
+    },
+    required: true,
+  },
+  isPaid: {
+    type: Boolean,
+    default: false,
+  },
+  timestamps: true,
```

Comment:
did you test it?

Reply:
it should work due to mongoose docs. We can change it later if it wont work

Reply:
I mean, that I saw it should be an object { timestamps: true } after these object with fields

Reply:
oh, yes